src/test/java/com/exasol/projectkeeper/ProjectKeeperIT.java

```diff
@@ -94,11 +96,11 @@ void testOnlyPomInvalid() throws IOException, InterruptedException {
         assertAll(//
                 () -> assertThat(result.getExitCode(), not(is(0))),
                 () -> assertThat(result.getStdout(),
-                        containsString("This projects structure does not conform with the template.")),
+                        containsString("E-PK-6 This projects structure does not conform with the template.")),
```

Comment:
Maybe you don't need to check the whole error message in case it doesn't contain any variables. I think for such cases just code is enough